packages/cli/package.json

```diff
@@ -12,7 +12,7 @@
     "@meeco/cryppo": "^0.9.0",
     "@meeco/file-storage-node": "2.1.1",
     "@meeco/keystore-api-sdk": "^0.14.0",
-    "@meeco/sdk": "^1.0.0",
+    "@meeco/sdk": "^2.0.0",
```

Comment:
probably we can have this be 2.0.0-beta until we've done some testing

Reply:
No problem